src/_data/team.yml

```diff
@@ -882,3 +882,13 @@
     <p>Outside of work he enjoys reading, travelling and generally having good food to eat. He likes to enjoy the finer things in life, given he wants to make the most out of it.</p>
   email: ade@codurance.com
   office: london
+
+- id: marius.alexandru.ignat
```

Comment:
The ID doesn't seem to be formatted properly. In the preview when you click on the image in the team page your bio doesn't appear, so I think the ID format is the cause.